flink-python/pyflink/table/expression.py

```diff
@@ -1558,6 +1558,18 @@ def map_keys(self) -> 'Expression':
         """
         return _unary_op("mapKeys")(self)
 
+    @property
+    def map_union(self, map2) -> 'Expression':
+        """
+        Returns a map created by merging two maps, 'map1' and 'map2'.
+        These two maps should have same data structure. If there are overlapping keys,
```

Comment:
it's not clear what is `same datastructure` in SQL context...
i guess same type

Reply:
ok, I will change the description

Reply:
`Returns a map created by merging two maps, 'map1' and 'map2'. These two maps should have same type. If there are overlapping keys, the value from 'map2' will overwrite the value from 'map1'. If any of maps is null, return null.`

What I want to say is that two maps' key and value should have same data type.

Reply:
you can not say `same` since  one could be nullable and another not...
so it should be common type

Reply:
`Returns a map created by merging two maps, 'map1' and 'map2'. These two maps are common map type. If there are overlapping keys, the value from 'map2' will overwrite the value from 'map1'. If any of maps is null, return null.

is this ok?

Reply:
I would change this sentence ` These two maps are common map type. ` in a way
` These two maps should have a common map type. `

Reply:
`Returns a map created by merging two maps, 'map1' and 'map2'. These two maps should have a common map type. If there are overlapping keys, the value from 'map2' will overwrite the value from 'map1'. If any of maps is null, return null.`